### DISCUSSION TITLE
Octobot connection to Nexo

### DISCUSSION THREAD

**Discussion by Virtruvian** [Drakkar-Software/OctoBot#2539] [Ideas]
Hey Guy's, Awesome job with Octobot. Im using it for a while now on Kucoin and works like a charm. 
I was wondering if there will be also support for Nexo. https://pro.nexo.com/
Cheers, Virtruvius

**Comment by Herklos** [comment]
Hi @Virtruvian!
Thanks for your feedback we really appreciate it.
Can you create a post [here](https://feedback.octobot.cloud/) so that we can track the interest of nexo for our roadmap please?
